From 784e4c108e037c5d02300f33bb3e6ccf46cd593e Mon Sep 17 00:00:00 2001
From: lrinQVQ <cqlrin@gmail.com>
Date: Wed, 20 Dec 2017 05:50:56 +0800
Subject: [PATCH 139/249] Switch RIL CAF HALs to staging/lineage-15.1-caf

reference-ril requirement qemu_pipe.h.
Switch to staging/lineage-15.1-caf for fix qemu_pipe.h include.

Change-Id: I6c1dba9d84053859c4735ee2b4ce905156a71245
---
 snippets/lineage.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/snippets/lineage.xml b/snippets/lineage.xml
index 56210f6..faff4d7 100644
--- a/snippets/lineage.xml
+++ b/snippets/lineage.xml
@@ -102,7 +102,7 @@
   <project path="hardware/qcom/media-caf/msm8996" name="LineageOS/android_hardware_qcom_media" groups="qcom,pdk" revision="lineage-15.0-caf-8996" />
   <project path="hardware/qcom/media-caf/msm8998" name="LineageOS/android_hardware_qcom_media" groups="qcom,pdk" revision="lineage-15.0-caf-8998" />
   <project path="hardware/qcom/wlan-caf" name="LineageOS/android_hardware_qcom_wlan" groups="qcom_wlan,pdk" revision="lineage-15.0-caf" />
-  <project path="hardware/ril-caf" name="LineageOS/android_hardware_ril" groups="pdk" revision="lineage-15.0-caf" />
+  <project path="hardware/ril-caf" name="LineageOS/android_hardware_ril" groups="pdk" revision="staging/lineage-15.1-caf" />
   <project path="system/qcom" name="LineageOS/android_system_qcom" groups="qcom" />
   <!--<project path="vendor/codeaurora/telephony" name="LineageOS/android_vendor_codeaurora_telephony" />-->
   <!--<project path="vendor/qcom/opensource/bluetooth" name="LineageOS/android_vendor_qcom_opensource_bluetooth" />-->
-- 
2.17.1

